cmd/application.go

```diff
@@ -167,6 +172,54 @@ var applicationSetCmd = &cobra.Command{
 	},
 }
 
+var applicationDescribeCmd = &cobra.Command{
+	Use:   "describe",
+	Short: "describe the given application",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := getOcClient()
+		fmt.Println("Application", args[0], " has:")
```

Comment:
I think `fmt.Println()` enters a whitespace anyway, so we don't need it before `has:`
Also, it'd be better to format this with `fmt.Printf()`, looks cleaner and like an entire sentence.